_studio/mfx_lib/encode_hw/h264/src/mfx_h264_encode_hw.cpp

```diff
@@ -759,9 +759,6 @@ mfxStatus ImplementationAvc::Init(mfxVideoParam * par)
         return MFX_WRN_PARTIAL_ACCELERATION;
 
     GUID guid;
-#if defined(MFX_ENABLE_MFE)
```

Comment:
@artem-shaporenko can you confirm that  this is safe change?

Reply:
It's ok for now.